frontend/src/pages/NewPipelineVersion.tsx

```diff
@@ -114,6 +115,7 @@ class NewPipelineVersion extends Page<{}, NewPipelineVersionState> {
   private _pipelineVersionNameRef = React.createRef<HTMLInputElement>();
   private _pipelineNameRef = React.createRef<HTMLInputElement>();
   private _pipelineDescriptionRef = React.createRef<HTMLInputElement>();
+  private _pipelineVersionDescriptionRef = React.createRef<HTMLInputElement>();
```

Comment:
Nit: this isn't used anywhere.
After a quick check, I think only the drop ref is actually used. The others were not used.